bin/check_samplesheet.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+# TODO nf-core: Update the script to check the samplesheet
```

Comment:
```suggestion
```

Reply:
Resolved in 05c2a0cfc8338bc3d6edf9fafe119438eb230589